src/main/java/uk/gov/hmcts/reform/divorce/casemaintenanceservice/service/impl/CcdAccessServiceImpl.java

```diff
@@ -101,8 +118,12 @@ private RespondentType validateLetterIdAndUserType(String letterHolderId, CaseDe
         }
         String respondentLetterHolderId = (String) caseDetails.getData().get(RESP_LETTER_HOLDER_ID_FIELD);
         String coRespondentLetterHolderId = (String) caseDetails.getData().get(CO_RESP_LETTER_HOLDER_ID_FIELD);
+        String isRespondentSolicitor = (String) caseDetails.getData().get(D8_RESP_SOLICITOR);
 
         if (letterHolderId.equals(respondentLetterHolderId)) {
+            if (StringUtils.equalsIgnoreCase(CmsConstants.YES_VALUE, isRespondentSolicitor)) {
```

Comment:
Minor: You don't need StringUtils for this comparation
```suggestion
            if (CmsConstants.YES_VALUE.equalsIgnoreCase(isRespondentSolicitor)) {
```

Reply:
Done 👍